protocols/MultiQCReport.sh

```diff
@@ -0,0 +1,114 @@
+#MOLGENIS nodes=1 ppn=1 mem=1gb walltime=03:00:00
+
+#Parameter mapping
+
+#string projectResultsDir
+#string project
+#string intermediateDir
+#string projectLogsDir
+#string projectQcDir
+#string scriptDir
+#list externalSampleID
+#string contact
+#string qcMatricsList
+#string gcPlotList
+#string seqType
+#string RVersion
+#string wkhtmltopdfVersion
+#string fastqcVersion
+#string samtoolsVersion
+#string picardVersion
+#string multiqcVersion
+#string anacondaVersion
+#string hisatVersion
+#string indexFileID
+#string ensembleReleaseVersion
+#string prepKit
+#string NGSRNAVersion
+#string groupname
+#string tmpName
+#string jdkVersion
+#string RVersion
+#string htseqVersion
+#string pythonVersion
+#string gatkVersion
+#string ghostscriptVersion
+#string kallistoVersion
+#string ensembleReleaseVersion
+
+
+#Function to check if array contains value
+array_contains () { 
+    local array="$1[@]"
+    local seeking=$2
+    local in=1
+    for element in "${!array-}"; do
+        if [[ "$element" == "$seeking" ]]; then
+            in=0
+            break
+        fi
+    done
+    return $in
+}
+
+cat > ${intermediateDir}/${project}_QC_config.yaml <<'_EOF'
+
+report_header_info:
```

Comment:
Also add costom logo and stuff (in the future).
details here
https://github.com/ewels/MultiQC/blob/master/multiqc_config_example.yaml#L93